x-pack/legacy/plugins/maps/public/layers/styles/vector/components/legend/style_property_legend_row.js

```diff
@@ -72,16 +72,17 @@ export class StylePropertyLegendRow extends Component {
     return !this.props.style.isDynamic() || !this.props.style.isComplete() || !this.props.style.getField().getName();
   }
 
-  _formatValue = value => {
+  _formatValue = (value, prefix) => {
     if (!this.state.hasLoadedFieldFormatter || !this._fieldValueFormatter || value === EMPTY_VALUE) {
       return value;
     }
 
-    return this._fieldValueFormatter(value);
+    return this.props.style.isFieldMetaEnabled()
+      ? `${prefix} ${this._fieldValueFormatter(value)}`
```

Comment:
I would _not_ show the prefix if the data isn't clamped by the range. so it wouldn't show `<0' if there's no data smaller than 0, or `>100` if no data is larger than 100.